src/styles/blocks/main.scss

```diff
@@ -0,0 +1,10 @@
+.main {
+  display: flex;
+  flex-wrap: wrap;
+  box-sizing: border-box;
+  justify-content: center;
+
+  margin: 0 auto;
+  padding-top: 50px;
+  width: 1000px;
```

Comment:
Why did you set the width 1000px? I think you're supposed to calculate width with formula (($card-width * 4) + (card-right-padding * 3))

Reply:
Yes, good idea, I will fix it